src/logger.js

```diff
@@ -6,7 +6,11 @@ function wrapConsole(logCommand) {
 
     args.unshift('[JS-BUY-SDK]: ');
     /* eslint-disable no-console */
-    console[logCommand](...args);
+    try {
+      Function.prototype.bind.call(console[logCommand], console).apply(console, args);
+    } catch (e) {
+      Function.prototype.bind.call(console.log, console).apply(console, args);
+    }
```

Comment:
I would do this once. Just define `log` at the top of the file, then call it in here. Doing a try/catch on every call is probably unnecessary.